src/vs/workbench/parts/comments/electron-browser/commentsEditorContribution.ts

```diff
@@ -602,7 +602,7 @@ registerThemingParticipant((theme, collector) => {
 			`}`);
 	}
 
-	let monacoEditorBackground = theme.getColor(peekViewEditorBackground);
+	let monacoEditorBackground = theme.getColor(peekViewTitleBackground);
```

Comment:
This looks better on most themes, but on Quiet Light it matches the rest of the background. Should we use something else like the input background? Or do you think it looks ok on Quiet Light?

Reply:
I noticed on some themes this does blend in. I thought about using the input background but then that would mean we'd need to use it in the text area, which currently uses `editorBackground` and was what I was trying to match.

Here's what it would look like if we used the input bg colo. Thoughts?
![image](https://user-images.githubusercontent.com/35271042/44229097-db613480-a14b-11e8-9494-e4fe3c6844b4.png)

![image](https://user-images.githubusercontent.com/35271042/44228974-87eee680-a14b-11e8-8200-71ba67e4c48f.png)

Reply:
That looks good to me 👍